cmd/jag/commands/firmware.go

```diff
@@ -97,8 +97,7 @@ func FirmwareUpdateCmd() *cobra.Command {
 				return err
 			}
 
-			// TODO(kasper): Make 'auto' the default.
-			if chip == "auto" {
+			if chip == "auto" || chip == "" {
```

Comment:
Do we still use 'auto' anywhere?

Reply:
I think it could be used as argument on the command-line.